upup/models/bindata.go

```diff
@@ -19193,7 +19193,13 @@ data:
         reload
         loop
         bind {{ KubeDNS.NodeLocalDNS.LocalIP }}
+        {{- if KubeDNS.NodeLocalDNS.KubeDnsOnly }}
```

Comment:
Instead of this, you can probably simplify the config by consolidating "." with the two sections above. I.e you only have "." and cluster domain in the end.

Reply:
Sure, I wanted to make "a small change" but I totally see your point